lib/connectors/gitlab/connector.rb

```diff
@@ -34,29 +33,29 @@ def display_name
 
       def configurable_fields
         @configurable_fields ||= {
-          :api_token => {
+          'api_token' => {
```

Comment:
I find symbols a bit easier to work with for constants than just strings. Why do we need to replace them?

Reply:
See my comment above: https://github.com/elastic/connectors/pull/159#discussion_r916761741